tests/topotests/ospf_multi_vrf_bgp_route_leak/r1/ospfd.conf

```diff
@@ -0,0 +1,27 @@
+!
+hostname r1
+password zebra
+log file /tmp/r1-ospfd.log
+!
+router ospf
+  ospf router-id 10.0.255.1
+  redistribute kernel
+  redistribute connected
+  redistribute static
+  network 10.0.1.0/24 area 0
```

Comment:
Suggested to use interface level enable ospf command

Reply:
I do prefer interface level commands, but with topotests, using the network command in the config file is a bit more convenient.